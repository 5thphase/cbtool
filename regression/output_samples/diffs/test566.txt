 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_94 (xxx.xxx.xxx.xxx )...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_95 (xxx.xxx.xxx.xxx )...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_93 (xxx.xxx.xxx.xxx )...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_96 (xxx.xxx.xxx.xxx )...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_92 (xxx.xxx.xxx.xxx )...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_98 (xxx.xxx.xxx.xxx )...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_97 (xxx.xxx.xxx.xxx )...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_99 (xxx.xxx.xxx.xxx )...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_100 (xxx.xxx.xxx.xxx )...
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725153 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725170 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725257 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725264 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725271 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725276 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725294 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725296 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725298 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725300 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725307 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |ibmderby_tradelite|1_x_client_tradelite->1_x_was |xxx.xxx.xxx.xxx |1412725312 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725318 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725323 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725342 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725347 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725354 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|Redis Version |2.8.4 
