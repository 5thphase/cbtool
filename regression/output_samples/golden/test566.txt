 status: Waiting for vm_95 (cloud-assigned uuid EFE82D82-D4D1-5B44-8B43-7D747CFFF14E) to start...
 status: Waiting for vm_93 (cloud-assigned uuid 0618D360-C583-5BDF-9CDE-91078F645356) to start...
 status: Waiting for vm_92 (cloud-assigned uuid 0EE066DB-E6F6-5B26-B4B1-91A657ED83D5) to start...
 status: Waiting for vm_94 (cloud-assigned uuid D1806A8A-0E6A-5B16-BEA4-B891BB6BB040) to start...
 status: Waiting for vm_96 (cloud-assigned uuid 6AB2BF65-B76C-56C7-B6C8-FC291C4DE3A6) to start...
 status: Trying to establish network connectivity to vm_95 (cloud-assigned uuid EFE82D82-D4D1-5B44-8B43-7D747CFFF14E), on IP address 193.117.31.228...
 status: Trying to establish network connectivity to vm_96 (cloud-assigned uuid 6AB2BF65-B76C-56C7-B6C8-FC291C4DE3A6), on IP address 205.203.109.228...
 status: Trying to establish network connectivity to vm_93 (cloud-assigned uuid 0618D360-C583-5BDF-9CDE-91078F645356), on IP address 56.206.10.12...
 status: Trying to establish network connectivity to vm_94 (cloud-assigned uuid D1806A8A-0E6A-5B16-BEA4-B891BB6BB040), on IP address 69.106.77.188...
 status: Trying to establish network connectivity to vm_92 (cloud-assigned uuid 0EE066DB-E6F6-5B26-B4B1-91A657ED83D5), on IP address 120.201.157.210...
 status: Bypassing the sending of a copy of the code tree to vm_95 (193.117.31.228)...
 status: Bypassing the sending of a copy of the code tree to vm_92 (120.201.157.210)...
 status: Bypassing the sending of a copy of the code tree to vm_94 (69.106.77.188)...
 status: Bypassing the sending of a copy of the code tree to vm_96 (205.203.109.228)...
 status: Bypassing the sending of a copy of the code tree to vm_93 (56.206.10.12)...
 status: Bypassing generic VM post_boot configuration on all VMs belonging to ai_23...
 status: Bypassing application-specific configuration on all VMs beloging to ai_23...
 status: QEMU Scraper will NOT be automatically started during the deployment of ai_23...
AI object E0D6F882-25F3-5DB1-B9AE-7E418698CE59 (named "ai_23") sucessfully attached to this experiment. It is ssh-accessible at the IP address 120.201.157.210 (0EE066DB-E6F6-5B26-B4B1-91A657ED83D5.simcloud.com).
 status: Waiting for vm_97 (cloud-assigned uuid CB2D60C5-E188-583D-94E4-0E697FC2619A) to start...
 status: Waiting for vm_98 (cloud-assigned uuid 446FC10E-D344-5FB4-89AA-6F9FF5623C1D) to start...
 status: Trying to establish network connectivity to vm_97 (cloud-assigned uuid CB2D60C5-E188-583D-94E4-0E697FC2619A), on IP address 223.21.179.66...
 status: Trying to establish network connectivity to vm_98 (cloud-assigned uuid 446FC10E-D344-5FB4-89AA-6F9FF5623C1D), on IP address 48.6.57.111...
 status: Bypassing the sending of a copy of the code tree to vm_98 (48.6.57.111)...
 status: Bypassing the sending of a copy of the code tree to vm_97 (223.21.179.66)...
 status: Bypassing generic VM post_boot configuration on all VMs belonging to ai_24...
 status: Bypassing application-specific configuration on all VMs beloging to ai_24...
 status: QEMU Scraper will NOT be automatically started during the deployment of ai_24...
AI object C82D8BC0-04DF-5341-9297-74C86F3133EC (named "ai_24") sucessfully attached to this experiment. It is ssh-accessible at the IP address 223.21.179.66 (CB2D60C5-E188-583D-94E4-0E697FC2619A.simcloud.com).
 status: Waiting for vm_101 (cloud-assigned uuid 6EBFFB82-2905-518D-8D8A-5D05D7712B16) to start...
 status: Waiting for vm_102 (cloud-assigned uuid 176E6D18-187C-5D91-ABED-A2DD992A34FB) to start...
 status: Waiting for vm_100 (cloud-assigned uuid C0FDF7E5-B6F7-5C87-9EDE-D989C94ECFC1) to start...
 status: Waiting for vm_99 (cloud-assigned uuid A90651DE-1CA3-5AB0-B48A-99E109641A45) to start...
 status: Trying to establish network connectivity to vm_102 (cloud-assigned uuid 176E6D18-187C-5D91-ABED-A2DD992A34FB), on IP address 107.12.67.52...
 status: Trying to establish network connectivity to vm_101 (cloud-assigned uuid 6EBFFB82-2905-518D-8D8A-5D05D7712B16), on IP address 204.126.84.249...
 status: Trying to establish network connectivity to vm_100 (cloud-assigned uuid C0FDF7E5-B6F7-5C87-9EDE-D989C94ECFC1), on IP address 108.226.73.41...
 status: Trying to establish network connectivity to vm_99 (cloud-assigned uuid A90651DE-1CA3-5AB0-B48A-99E109641A45), on IP address 226.78.129.208...
 status: Bypassing the sending of a copy of the code tree to vm_102 (107.12.67.52)...
 status: Bypassing the sending of a copy of the code tree to vm_100 (108.226.73.41)...
 status: Bypassing the sending of a copy of the code tree to vm_99 (226.78.129.208)...
 status: Bypassing the sending of a copy of the code tree to vm_101 (204.126.84.249)...
 status: Bypassing generic VM post_boot configuration on all VMs belonging to ai_25...
 status: Bypassing application-specific configuration on all VMs beloging to ai_25...
 status: QEMU Scraper will NOT be automatically started during the deployment of ai_25...
AI object 1C12610E-34ED-53E9-8995-3A01284A4C35 (named "ai_25") sucessfully attached to this experiment. It is ssh-accessible at the IP address 226.78.129.208 (A90651DE-1CA3-5AB0-B48A-99E109641A45.simcloud.com).
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |231.86.90.94    |1381326903     |none                                    |37DC0C46-B289-52BE-B23A-838CCC59A153
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |128.1.118.199   |1381326922     |none                                    |817F3716-E289-5E74-9CFA-439D68B796FF
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |40.226.122.43   |1381327012     |none                                    |BC3E5204-4D7F-55E4-89A9-76B4E6ADE788
|ai_9      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |76.45.10.114    |1381327017     |none                                    |7EC593E7-D1D6-5AE6-95C1-26A50C79CA8D
|ai_10     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |129.104.39.58   |1381327023     |none                                    |8D553CE9-062A-5B54-A4FE-1C25F96EF9F0
|ai_11     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |245.212.146.66  |1381327029     |none                                    |24A22ADF-E4E2-5A57-B3C9-715E0E55A9A9
|ai_14     |netperf        |1_x_netclient->1_x_netserver                      |45.225.174.253  |1381327048     |none                                    |9A561230-8C17-5CA7-975F-B38D11D1B55A
|ai_15     |netperf        |1_x_netclient->1_x_netserver                      |169.3.225.23    |1381327049     |none                                    |38F14D52-8CF7-52D9-BAC0-0995E730B892
|ai_16     |netperf        |1_x_netclient->1_x_netserver                      |88.52.88.210    |1381327051     |none                                    |570CE098-C661-5AA3-9398-74E51A38D8E6
|ai_17     |netperf        |1_x_netclient->1_x_netserver                      |20.154.37.35    |1381327054     |none                                    |ACC2FFBF-1C0D-50A6-A8CB-CCC00B3BD88A
|ai_18     |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |250.109.114.117 |1381327061     |none                                    |D85EFE4C-74D6-5476-9FE7-617FE9F815AA
|ai_19     |tradelite      |1_x_client_tradelite->1_x_was                     |149.13.13.8     |1381327067     |none                                    |B2F4C12B-BF88-5075-839A-87D687E189DE
|ai_20     |hpcc           |1_x_fen_hpc->3_x_cn_hpc                           |252.20.69.139   |1381327073     |none                                    |49EF672C-C239-50C6-A92F-49B050CD0A24
|ai_21     |coremark       |1_x_coremark                                      |69.63.119.137   |1381327078     |none                                    |5D2AC228-2FC0-5650-A023-720E9D761150
|ai_23     |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |120.201.157.210 |1381327098     |none                                    |E0D6F882-25F3-5DB1-B9AE-7E418698CE59
|ai_24     |netperf        |1_x_netclient->1_x_netserver                      |223.21.179.66   |1381327103     |none                                    |C82D8BC0-04DF-5341-9297-74C86F3133EC
|ai_25     |hpcc           |1_x_fen_hpc->3_x_cn_hpc                           |226.78.129.208  |1381327108     |none                                    |1C12610E-34ED-53E9-8995-3A01284A4C35

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_36         |client_daytrader    |nano32      |231.86.90.94    |simhostd1                  |LG                  |ai_4    |none       |D74FF3C7-0748-5DF4-95AA-3C5CA991145F
|vm_37         |was                 |iron32      |28.52.95.113    |simhostc1                  |SUT                 |ai_4    |none       |3DCEBD35-7918-5ECA-B5BE-754974C9551D
|vm_38         |db2                 |platinum64  |237.245.160.7   |simhosta1                  |SUT                 |ai_4    |none       |F7D8E555-DA9B-5B14-ABF6-21B93BC4BD78
|vm_39         |netserver           |iron32      |176.49.35.238   |simhostc1                  |SUT                 |ai_5    |none       |B429FD85-77F3-5CDA-BDA3-BE6BAE29F359
|vm_40         |netclient           |platinum64  |128.1.118.199   |simhosta1                  |SUT                 |ai_5    |none       |44CD170B-2DBF-5057-A049-F0158C7B2C65
|vm_48         |was                 |iron32      |215.98.31.84    |simhosta1                  |SUT                 |ai_8    |none       |B2243313-7D32-5191-B701-8CFCBD90929B
|vm_47         |was                 |iron32      |132.111.246.150 |simhosta1                  |SUT                 |ai_8    |none       |AA4D7546-6AE4-5C37-BF8E-8F738DA6DDA2
|vm_50         |lb                  |iron32      |42.85.220.153   |simhosta1                  |SUT                 |ai_8    |none       |4F969A74-1BA5-5159-82A5-3F3A920FAE16
|vm_49         |client_daytrader    |nano32      |40.226.122.43   |simhostd1                  |LG                  |ai_8    |none       |A9853882-9AED-5574-B00F-47E9E6901979
|vm_51         |db2                 |gold32      |28.120.188.56   |simhostb1                  |SUT                 |ai_8    |none       |FD8C8D85-297F-5F1B-BECF-097657CB1F3F
|vm_52         |client_daytrader    |nano32      |76.45.10.114    |simhostd1                  |LG                  |ai_9    |none       |7912B8BF-2F10-5009-B993-0975B5F493CB
|vm_53         |was                 |iron32      |44.65.231.70    |simhostc1                  |SUT                 |ai_9    |none       |E21DF9A9-46A8-507D-8FA2-FCABA0A92EB9
|vm_54         |db2                 |platinum64  |136.72.89.49    |simhostb1                  |SUT                 |ai_9    |none       |9066E1E4-6EF0-524C-BAD8-2AC982900577
|vm_55         |client_daytrader    |nano32      |129.104.39.58   |simhostd1                  |LG                  |ai_10   |none       |D6F5AFA3-C392-5608-8A99-26EBC9039489
|vm_56         |db2                 |platinum64  |44.90.22.168    |simhosta1                  |SUT                 |ai_10   |none       |DEF06AE2-0B9F-5D16-BF96-5BA332CBCB94
|vm_57         |was                 |iron32      |175.16.7.112    |simhosta1                  |SUT                 |ai_10   |none       |52D9A1F2-1962-5DD5-A39B-89F1D158C570
|vm_58         |was                 |iron32      |210.138.249.161 |simhosta1                  |SUT                 |ai_11   |none       |EF610B97-06D3-5306-8CFD-BC4CE6BAC187
|vm_60         |db2                 |platinum64  |111.189.220.23  |simhostb1                  |SUT                 |ai_11   |none       |632229C9-E95F-52E2-8C6E-C07D7F58A933
|vm_59         |client_daytrader    |nano32      |245.212.146.66  |simhostd1                  |LG                  |ai_11   |none       |4610480A-9A66-50D3-A669-06BD44FDAD25
|vm_71         |netclient           |iron32      |45.225.174.253  |simhostd1                  |LG                  |ai_14   |none       |2A2DE178-ECF0-591A-86E7-14D968D5B989
|vm_72         |netserver           |iron32      |105.252.200.86  |simhostc1                  |SUT                 |ai_14   |none       |2D57E16C-027B-5810-B54F-4D617F039E23
|vm_73         |netclient           |iron32      |169.3.225.23    |simhostd1                  |LG                  |ai_15   |none       |8FD4E23F-F527-5E23-A168-C675C5A0DACF
|vm_74         |netserver           |iron32      |32.221.108.241  |simhostb1                  |SUT                 |ai_15   |none       |C71192EB-1952-5A0C-8BCE-080EE58E187E
|vm_75         |netclient           |iron32      |88.52.88.210    |simhostd1                  |LG                  |ai_16   |none       |FBBB1A42-8705-59EA-9C36-1D64BD54B245
|vm_76         |netserver           |iron32      |203.132.223.71  |simhostb1                  |SUT                 |ai_16   |none       |C9DF5A34-F657-522A-B2F9-A4AF1E3F0588
|vm_77         |netclient           |iron32      |20.154.37.35    |simhostd1                  |LG                  |ai_17   |none       |3D32B086-0C55-5812-96E9-6F065EF36E83
|vm_78         |netserver           |iron32      |50.153.89.20    |simhosta1                  |SUT                 |ai_17   |none       |FD60F101-9E95-5955-9F1B-B95755AD37AC
|vm_79         |client_daytrader    |nano32      |250.109.114.117 |simhostd1                  |LG                  |ai_18   |none       |D385E591-0CA0-5BBB-B38A-D47A795A375A
|vm_80         |was                 |iron32      |81.102.228.127  |simhostc1                  |SUT                 |ai_18   |none       |0330F4AA-1995-594D-B752-60465210F725
|vm_83         |db2                 |gold32      |78.148.218.186  |simhostc1                  |SUT                 |ai_18   |none       |FDF8006D-6C26-5188-9648-A7685AD98308
|vm_81         |was                 |iron32      |87.175.39.32    |simhostb1                  |SUT                 |ai_18   |none       |ABC48D2D-07CD-5EA0-9935-FDF4B1C1785A
|vm_82         |lb                  |iron32      |22.74.18.8      |simhostb1                  |SUT                 |ai_18   |none       |94655AE6-C487-51F7-B915-F0E257A61987
|vm_84         |was                 |iron32      |105.104.60.242  |simhostc1                  |SUT                 |ai_19   |none       |778D0515-E58B-5096-9261-11336816160A
|vm_85         |client_tradelite    |nano32      |149.13.13.8     |simhostd1                  |LG                  |ai_19   |none       |B8E92BB3-78E9-5E57-9DAF-E67340BEC8F7
|vm_86         |fen_hpc             |bronze32    |252.20.69.139   |simhostb1                  |SUT                 |ai_20   |none       |9D5E503E-A7E2-522D-BCBC-E44CA3722CAA
|vm_87         |cn_hpc              |bronze32    |136.40.139.114  |simhosta1                  |SUT                 |ai_20   |none       |E7C5E961-1E2D-511D-BD7E-4D8434C6EB80
|vm_88         |cn_hpc              |bronze32    |179.109.53.201  |simhostc1                  |SUT                 |ai_20   |none       |01DFEB9F-8BD5-5B7B-9113-B54EBD6C7FCD
|vm_89         |cn_hpc              |bronze32    |176.116.56.20   |simhostc1                  |SUT                 |ai_20   |none       |DC82504D-C0A3-5FC4-93CE-728B952D2ACE
|vm_90         |coremark            |iron32      |69.63.119.137   |simhostc1                  |SUT                 |ai_21   |none       |0A97CB97-8AD8-5EB2-B631-8CA8602EB367
|vm_92         |client_daytrader    |nano32      |120.201.157.210 |simhostd1                  |LG                  |ai_23   |none       |49785AA0-8A0E-55BE-B8EF-361E1E5C8210
|vm_93         |lb                  |iron32      |56.206.10.12    |simhostc1                  |SUT                 |ai_23   |none       |2B47E770-D574-5754-87A8-59196927DA18
|vm_94         |db2                 |gold32      |69.106.77.188   |simhostc1                  |SUT                 |ai_23   |none       |E3DE82AF-3EA9-56D2-8864-6BC8A73F7910
|vm_95         |was                 |iron32      |193.117.31.228  |simhostc1                  |SUT                 |ai_23   |none       |465E8E0A-0D6E-5746-98B2-FC376C6C6BDA
|vm_96         |was                 |iron32      |205.203.109.228 |simhostb1                  |SUT                 |ai_23   |none       |FC2BE9BD-D543-5E37-A5A3-9912CCEBFB0B
|vm_97         |netclient           |iron32      |223.21.179.66   |simhostd1                  |LG                  |ai_24   |none       |95370F58-45B1-5632-85EF-C8D72F17C9B9
|vm_98         |netserver           |iron32      |48.6.57.111     |simhostc1                  |SUT                 |ai_24   |none       |F8441AB6-6648-57E4-BA58-21DD240841D1
|vm_99         |fen_hpc             |bronze32    |226.78.129.208  |simhostc1                  |SUT                 |ai_25   |none       |9C786819-47B0-5CFD-BA8E-31DEB6BC0F7D
|vm_100        |cn_hpc              |bronze32    |108.226.73.41   |simhosta1                  |SUT                 |ai_25   |none       |D5FEC27A-73D5-5EA2-9658-EC6869CC3445
|vm_102        |cn_hpc              |bronze32    |107.12.67.52    |simhostb1                  |SUT                 |ai_25   |none       |0C2D9527-B30B-5CC8-BDD2-A372E493177B
|vm_101        |cn_hpc              |bronze32    |204.126.84.249  |simhosta1                  |SUT                 |ai_25   |none       |C45F9127-1A15-573F-9BA5-62626BE3833F

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |5.96M                         
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |2417715                       
|Total Connections Received                           |4604                          
|Total Commands Processed                             |4026434                       
|Number of Keys                                       |895                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |5449940992.0                  
|Data Size                                            |4962802376.0                  
|Index Size                                           |133824768                     
|Average Object Size                                  |8099.53270543                 
|Collections                                          |40                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |50                            
|AIs                                                  |17                            
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |50                            
|VMs ARRIVED                                          |99                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |49                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |17                            
|AIs ARRIVED                                          |25                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |8                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |272                           

 ---------------- VMC-WIDE COUNTERS ----------------
|CEA17C37-820A-507F-9356-90956267A568 (simzone_a) VM RESERVATIONS|12                            
|966D86CB-90BB-553D-A06A-0FFA0818D568 (simzone_d) VM RESERVATIONS|13                            
|C41BC369-F411-5382-B95F-70C31BD20E23 (simzone_c) VM RESERVATIONS|15                            
|2FF9728E-95B3-5EE9-A0A4-61875B507FA8 (simzone_b) VM RESERVATIONS|10                            

